api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/LRA.java

```diff
@@ -241,38 +241,48 @@
 
     /**
      * <p>
-     * Normally if an LRA is present when a bean method is executed it will not
-     * be ended when the method returns. To override this behaviour and force LRA
-     * termination on exit use the end element.
+     * Normally if an LRA is present when a resource method is executed it will not
+     * be ended when the method returns. To override this behaviour and force the LRA
+     * to close on exit use this <code>end</code> element with the value set to
+     * {@literal true}.
      * </p>
      *
      * <p>
-     * If the <code>end</code> value is set to <code>false</code>
-     * while the annotated method returns the {@link Response} status code
-     * with the meaning the LRA to be cancelled (see {@link #cancelOn()} and {@link #cancelOnFamily()})
-     * then this error state has precedence over the <code>end</code> attribute.
-     * It means the LRA will be cancelled despite the end was set to <code>false</code>.
+     * If the <code>end</code> value is set to {@literal false}
+     * but the annotated method finishes with a status code that matches
+     * one of the {@link #cancelOn()} and {@link #cancelOnFamily()} values then
```

Comment:
```suggestion
     * one of the {@link #cancelOn()} or {@link #cancelOnFamily()} values then
```

Reply:
A pointless comment. But I'd prefer to stay with my text.

Reply:
for me it is either one of cancelOn or cancelOnFamily, "and" just doesn't fit here, but ok

Reply:
Try that last commit (it rewords the text and also uses the word `or`)